stable/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -64,7 +64,13 @@ spec:
           - sh
           - -c
           - |
-            chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} /bitnami
+            set -e
+            for ref in $(find /bitnami -mindepth 1 -maxdepth 1 -not -name ".snapshot"); do
+              chown {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} $ref
+            done
+            for ref in $(find /bitnami -mindepth 2 -maxdepth 2 -not -name ".snapshot"); do
+              chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} $ref
+            done
```

Comment:
Why not using a single line?
```bash
find {{ .Values.persistence.mountPath }} -mindepth 1 -maxdepth 1 -not -name ".snapshot" | xargs chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }}
```

Reply:
thanks for the swift reply!

imho, iteration over `$ref` in for loop is more readable, and safer than `xargs`, since some version of xargs do not deal well with empty set (from experience)

Reply:
Hi @anapsix 

Currently, these templates are already pretty complex. We support many combinations depending on parameters and, as a result, these templates are becoming quite big (for instance, the `statefulset.yaml` has already more than 300 lines) and that make it really hard to understand.

That's why I think we should try to avoid to use 6 lines of code when can use just one.

On the other hand, from a "performance" point of view, `xargs` is able to iterate over each of the results much faster.

Reply:
hey @juan131, how about just using find (see latest commit)?

Reply:
Why not using just one `find` call?
```
find {{ .Values.persistence.mountPath }} -mindepth 1 -maxdepth 1 -not -name ".snapshot" \
              -exec chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} {} \;
```
You don't need the 1st one if you're using `{{ .Values.persistence.mountPath }}` as the directory to find and `-mindepth 1 -maxdepth 1`

Reply:
I agree, using `find` on `{{ .Values.persistence.mountPath }}` would work just fine..
I've used two `find` statements, since I was attempting to remain as close to original functionality as possible. First, to ensure everything in `/bitnami` is "chowned", and second - to handle mounted PV.
If you don't see the need for setting ownership on `/bitnami/*`, I'm all for it.
In fact, updated to use `xargs` with single `find` command, to make sure script fails if `chown` is unsuccessful (since find's "-exec" eats failures)